docs/src/guides/laravel/deploy/scheduling.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Workers, Cron Jobs and Task Scheduling"
+sidebarTitle: "Scheduling Tasks"
+weight: -100
+description: How to schedule Tasks for your Laravel application.
+---
+
+Laravel offers a very convenient and flexible way of scheduling tasks. A large set of [helpers functions](https://laravel.com/docs/scheduling#schedule-frequency-options) allows the easy scheduling of Commands and Jobs.
+
+Once the scheduled tasks are defined, they need to be effectively executed at the right time and pace.
+The recommended way is a cron configuration entry running the `artisan schedule:run` command.
+
+```yaml {location=".platform.app.yaml"}
+crons:
+    # Run Laravel's scheduler every 5 minutes, which is as often as crons can run.
+    scheduler:
+        spec: '*/5 * * * *'
+        cmd: 'php artisan schedule:run'
+```
+
+The [minimum time between cron jobs](../../../configuration/app/app-reference.html#cron-job-timing) being triggered depends on your plan. The tasks scheduling may then be contradicted by the cron minimum frequency.
```

Comment:
```suggestion
The [minimum time between cron jobs](../../../configuration/app/app-reference.html#cron-job-timing) being triggered depends on your plan. Task scheduling may then be contradicted by the cron minimum frequency.
```

For the second sentence, what's the result of this contradiction? Does it mean that the crons then won't run? It might help to clarify it a little. At least for me, I wasn't immediately sure what the contradiction would mean in practice.

Reply:
The cron will run, but the scheduled tasks won't be triggered. "Out-of-beat" schedules will be ignored. Thanks for raising this. Let me update the comment.